standalone/main.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2020-2021 Pilz GmbH & Co. KG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+#include <chrono>
+#include <sstream>
+#include <string>
+#include <thread>
+
+#include <psen_scan_v2_standalone/core.h>
+
+using namespace psen_scan_v2_standalone;
+using namespace psen_scan_v2_standalone::constants;
+
+/*
+ * In this section we declare all necessary configuration parameters
+ */
+const std::string HOST_IP{ "192.168.0.50" };
+const std::string SCANNER_IP{ "192.168.0.10" };
+// Start- and end-angle have been configured to be in the middle of the scan range.
+const TenthOfDegree ANGLE_START{ degreeToTenthDegree(137) };
+const TenthOfDegree ANGLE_END{ degreeToTenthDegree(138) };
+
+/*
+ * This function is used as a callback every time a new laserscan is received.
+ */
+void laserScanCallback(const LaserScan& scan)
+{
+  const LaserScan::MeasurementData& measures = scan.getMeasurements();
+
+  std::stringstream strstr;
+  for (auto it = measures.cbegin(); it < measures.cend(); ++it)
+  {
+    strstr << " " << *it;
+  }
+  PSENSCAN_INFO_THROTTLE(1 /* sec */, "laserScanCallback()", strstr.str());
```

Comment:
On Ubuntu focal (20.04) the values do not appear. However without `_THROTTLE` it works!